src/org/omegat/ColorScheme_dark.properties

```
@@ -0,0 +1,40 @@
+# OmegaT application default colors scheme
+# This is a table of default colors definitions when no customization.
+# Here is a definition for dark theme, such as GTK LAF(dark), OSX Mojava and 3rd party dark LAFs.
+OmegaT.activeSource=28a028
+OmegaT.source=a0cfa0
+OmegaT.noted=a0c0c0
+OmegaT.untranslated=4c874c
+OmegaT.translated=afaf49
+OmegaT.nonUnique=808080
+OmegaT.placeholder=969696
+OmegaT.removeTextTarget=ff0000
+OmegaT.NBSP=c8c8c8
+OmegaT.whiteSpace=808080
+OmegaT.bidiMarkers=c80000
+OmegaT.paragraphStart=aeaeae
+OmegaT.markComesFromTM=fa8072
+OmegaT.markComesFromTMXice=af76df
+OmegaT.markComesFromTMX100PC=ff9408
+OmegaT.markComesFromTMXAuto=ffd596
+OmegaT.markComesFromTMXenForced=ffccff
```

These are a bit confusing because it looks like they are about TMX. Could we change them to:

```suggestion
OmegaT.markComesFromTmXice=af76df
OmegaT.markComesFromTmX100pc=ff9408
OmegaT.markComesFromTmXauto=ffd596
OmegaT.markComesFromTmXenforced=ffccff
```

Then for consistency `OmegaT.NBSP` should become `OmegaT.nbsp`.